features/netsocket/CellularInterface.h

```diff
@@ -23,6 +23,8 @@
  *  Common interface that is shared between Cellular interfaces
```

Comment:
This should be functionally the same but based on UART rather than USB. I'd expect unsupported version to be deleted once this lands